src/solver/conv_asm_implicit_gemm_gtc_fwd_nhwc.cpp

```diff
@@ -583,6 +583,10 @@ bool ConvAsmImplicitGemmGTCDynamicFwdXdlopsNHWC::IsApplicable(const ConvolutionC
 
     if(!ctx.IsLayoutNHWC())
         return false;
+
+    const auto target = ctx.GetStream().GetTargetProperties();
+    if(target.Xnack() && *target.Xnack())
+        return false;
```

Comment:
I got a feeling (unproven) that if you add an empty line after line 589, clang tidy will stop complaining [readability-simplify-boolean-expr]

Reply:
It wants `return !(target.Xnack() && *target.Xnack())` but this is :shit:

Reply:
I recommend shutting it up.

Reply:
It did not complain other files, so I guess its static code check got confused with the 
>    if(target.Xnack() && *target.Xnack())
        return false;
return true;

maybe just change it to
>    if(target.Xnack() && *target.Xnack())
        return false;
<--- empty line here ---> (non proven WA)
>return true;

Reply:
Thank you! Fixed in 0da6aa2